protos/world.proto

```diff
@@ -10,4 +10,10 @@ message GetAirbasesRequest {
 
 message GetAirbasesResponse {
   repeated Airbase airbases = 1;
+}
+
+message GetMarkPanelsRequest {}
+
+message GetMarkPanelsResponse {
+	repeated MarkPanel markPanels = 1;
```

Comment:
```suggestion
	repeated MarkPanel mark_panels = 1;
```

Reply:
My C# muscle memory is beating out by Ruby muscle memory. Oh no...

Changed.